src/algebra/group/defs.lean

```diff
@@ -20,8 +20,37 @@ The file does not contain any lemmas except for
 * lemmas required for instances.
 
 For basic lemmas about these classes see `algebra.group.basic`.
+
+We also introduce notation classes `has_scalar` and `has_vadd` for multiplicative and additive
+actions and register the following instances:
+
+- `has_pow M ℕ`, for monoids `M`, and `has_pow G ℤ` for groups `G`;
+- `has_scalar ℕ M` for additive monoids `M`, and `has_scalar ℤ G` for additive groups `G`.
+
+## Notation
+
+- `+`, `-`, `*`, `/`, `^` : the usual arithmetic operations; the underlying functions are
+  `has_add.add`, `has_neg.neg`/`has_sub.sub`, `has_mul.mul`, `has_div.div`, and `has_pow.pow`.
+- `a • b` is used as notation for `has_scalar.smul a b`.
+- `a +ᵥ b` is used as notation for `has_vadd.vadd a b`.
```

Comment:
Down below there's the sentence

> Nice notation and a basic theory for the power function on monoids and the `ℕ`-action on additive
monoids are built in the file `algebra.group_power.basic`. For now, we only register the most basic
properties that we need right away.

This is out of date, the nice notation is now immediate

Reply:
Fixed